src/utils/reactionCleanupTracking.ts

```diff
@@ -1,4 +1,41 @@
 import { Reaction } from "mobx"
+import { FinalizationRegistry as FinalizationRegistryMaybeUndefined } from "./FinalizationRegistryWrapper"
+
+const {
+    addReactionToTrack,
+    recordReactionAsCommitted,
+    resetCleanupScheduleForTests,
+    forceCleanupTimerToRunNowForTests
+} = FinalizationRegistryMaybeUndefined
+    ? createReactionCleanupTrackingUsingFinalizationRegister(FinalizationRegistryMaybeUndefined)
```

Comment:
It would be nice to separate each of these methods into a separate file. For easier maintenance in case, we want to drop timers eventually.

Reply:
@Bnaya After splitting this I think we merge :)